tests/sentry/relay/test_config.py

```diff
@@ -600,13 +600,7 @@ def test_project_config_with_span_attributes(default_project, insta_snapshot):
     # The span attributes config is not set with the flag turnd off
     cfg = get_project_config(default_project, full_config=True)
     cfg = cfg.to_dict()
-    assert "spanAttributes" not in cfg["config"]
-
-    with Feature("projects:performance-suspect-spans-ingestion"):
-        cfg = get_project_config(default_project, full_config=True)
-
-    cfg = cfg.to_dict()
-    insta_snapshot(cfg["config"]["spanAttributes"])
+    assert "spanAttributes" in cfg["config"]
```

Comment:
Should we restore the `insta_snapshot` from line 609 here?

Reply:
oh we should, I think that was a merge artifact I missed, one sec